### DISCUSSION TITLE
Welcome to PowerApps Discussions!

### DISCUSSION THREAD

**Discussion by aprildunnam** [aprildunnam/PowerApps#84] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MikeBetts44** [comment]
Hi, has anybody created  near miss or close call powerapp for construction projects? This should have features such as categorisation, description, image / video capture.

**Comment by adenmaigag** [comment]
The solution file is invalid. The compressed file must contain the following files at its root: solution.xml, customizations.xml, and [Content_Types].xml. Customization files exported from previous versions of Microsoft Dynamics 365 are not supported.

**Comment by joefyun** [comment]
Hi April, I really like this Room Reservation App but I wanted to know if it can pull up a building, floor, and room data from SQL as well as SVG floor plans? Please advise.

**Comment by Boredomder** [comment]
@aprildunnam Thank you so much, the app works well. however, I have observed that reserving the desk for a particular time slot during the day will prevent it from being available for selection during other time slots on the same day, could you please advise?

Here is your syntax:

Filter(
    Sort(col_Desks,Title),
    Not(
        Title in col_Reservations.DeskText
       ) && Active = 1 && Or(DeskSearchFilter.Text in Title , DeskSearchFilter.Text in Description) 

)
